upload/catalog/controller/checkout/payment_method.php

```diff
@@ -57,7 +57,7 @@ public function index(): string {
 			if (isset($this->session->data['payment_methods'])) {
 				$data['payment_methods'] = $this->session->data['payment_methods'];
 			} else {
-				if (!isset($this->session->data['payment_address'])) {
+				if (!empty($this->session->data['payment_address'])) {
```

Comment:
It's already changed to isset for the shipping_address from !isset in the commits. !empty would not be necessary in this case.